src/Select/Select.d.ts

```diff
@@ -12,6 +13,7 @@ export interface SelectProps extends StandardProps<InputProps, SelectClassKey, '
   native?: boolean;
   onClose?: (event: React.ChangeEvent<{}>) => void;
   onOpen?: (event: React.ChangeEvent<{}>) => void;
+  onChange: SelectInputProps['onChange'];
```

Comment:
This makes the Select's `onChange` prop required even though the SelectInput's `onChange` prop is optional. If we want the types to match, then I think we should be using `Pick`:
```ts
export interface SelectProps
  extends StandardProps<InputProps, SelectClassKey, 'value' | 'onChange'>,
    Pick<SelectInputProps, 'onChange'> {
  autoWidth?: boolean;
  ...
  // don't manually define onChange in here
}
```

I know this has already been merged, so sorry if I'm misunderstanding the purpose of this.

Reply:
Fair enough, I think we can make `onChange` optional as well but your solution also looks nice.

Reply:
ok thanks for the quick reply 😄. I'll make a new PR as a follow-up.